django_vite/templatetags/django_vite.py

```diff
@@ -38,19 +38,17 @@
 DJANGO_VITE_ASSETS_PATH = getattr(settings, "DJANGO_VITE_ASSETS_PATH")
 
 # Prefix for STATIC_URL
-DJANGO_VITE_STATIC_URL_PREFIX = getattr(
-    settings, "DJANGO_VITE_STATIC_URL_PREFIX", ""
-)
+DJANGO_VITE_STATIC_URL_PREFIX = getattr(settings, "DJANGO_VITE_STATIC_URL_PREFIX", "")
```

Comment:
This is just _Black_ formatting. I could remove it if it bothers you.

Reply:
No thanks !
I plan to introduce Black and Flake8 with pre-commit and Github Actions as there is more people contributing to the project.